README.md

```diff
@@ -1,12 +1,13 @@
 <p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_200.png" alt="Resoto"/>
-<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1></p>
+<h1 align="center">Automate tedious infrastructure tasks, remarkably fast!</h1>
 
-<p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_banner.png" /></p>
+<p align="center"><img src="https://raw.githubusercontent.com/someengineering/resoto/main/misc/resoto_banner.png" alt="Banner"/></p>
```

Comment:
It's actually best practice to omit the alt tag for decorative images.

Reply:
My linter complains - nevertheless removed it.